src/backoffice/dashboards/examine-management/views/section-view-examine-indexers.ts

```diff
@@ -0,0 +1,224 @@
+			this._modalService = modalService;
+		});
+	}
+
+	connectedCallback(): void {
+		super.connectedCallback();
+		this._getIndexData();
+	}
+
+	private async _onRebuildHandler() {
+		const modalHandler = this._modalService?.confirm({
+			headline: `Rebuild ${this.indexName}`,
+			content: html`
+				This will cause the index to be rebuilt.<br />
+				Depending on how much content there is in your site this could take a while.<br />
+				It is not recommended to rebuild an index during times of high website traffic or when editors are editing
+				content.
+			`,
+			color: 'danger',
+			confirmLabel: 'Rebuild',
+		});
+		modalHandler?.onClose().then(({ confirmed }) => {
+			if (confirmed) this._rebuild();
+		});
+	}
+	private async _rebuild() {
+		this._buttonState = 'waiting';
+		if (this._indexData.name)
+			try {
+				await SearchResource.postSearchIndexByIndexNameRebuild({ indexName: this._indexData.name });
+				this._buttonState = 'success';
+			} catch (e) {
+				this._buttonState = 'failed';
+				if (e instanceof ApiError) {
+					const error = e as ProblemDetails;
+					const data: UmbNotificationDefaultData = { message: error.message ?? 'Rebuild error' };
+					this._notificationService?.peek('danger', { data });
+				}
+			}
+	}
+
+	render() {
+		if (this._indexData) {
+			return html` <uui-box headline="${this.indexName}">
+					<p>
+						<strong>Health Status</strong><br />
+						The health status of the ${this._indexData.name} and if it can be read
+					</p>
+					<div>
+						<uui-icon-essentials>
+							<uui-icon
+								name=${this._indexData.isHealthy ? `check` : `wrong`}
+								class=${this._indexData.isHealthy ? 'positive' : 'danger'}>
+							</uui-icon>
+						</uui-icon-essentials>
+						${this._indexData.healthStatus}
+					</div>
+				</uui-box>
+				<umb-dashboard-examine-searcher searcherName="${this.indexName}"></umb-dashboard-examine-searcher>
+				${this.renderPropertyList()} ${this.renderTools()}`;
+		} else return html``;
+	}
+
+	private renderPropertyList() {
+		return html`<uui-box headline="Index info">
+			<p>Lists the properties of the ${this._indexData.name}</p>
+			<uui-table class="info">
+				<uui-table-row>
+					<uui-table-cell style="width:0px; font-weight: bold;"> documentCount </uui-table-cell>
+					<uui-table-cell>${this._indexData.documentCount} </uui-table-cell>
+				</uui-table-row>
+				<uui-table-row>
+					<uui-table-cell style="width:0px; font-weight: bold;"> fieldCount </uui-table-cell>
+					<uui-table-cell>${this._indexData.fieldCount} </uui-table-cell>
+				</uui-table-row>
+				${this._indexData.providerProperties
+					? Object.entries(this._indexData.providerProperties).map((entry) => {
+							return html`<uui-table-row>
+								<uui-table-cell style="width:0px; font-weight: bold;"> ${entry[0]} </uui-table-cell>
+								<uui-table-cell clip-text> ${JSON.stringify(entry[1]).replace(/,/g, ', ')} </uui-table-cell>
```

Comment:
I think it works a lot better by just printing out the value, which will inherently invoke its own `toString` function.

```suggestion
								<uui-table-cell clip-text> ${entry[1]} </uui-table-cell>
```

Example where boolean values are still there and strings don't have quotes:

![image](https://user-images.githubusercontent.com/752371/202708141-1031a70a-2159-4ce6-87cb-baede1be44fb.png)